### ISSUE TITLE
Padded block rules has false positive when block come after template string

### ISSUE THREAD

**Issue by soda0289** [eslint/typescript-eslint-parser#255]
**What version of TypeScript are you using?**
2.2.2

**What version of `typescript-eslint-parser` are you using?**
master

**Eslint Version**
master

**ESLint config**
```js
//    "extends": ["eslint:all"],
    "parser": "typescript-eslint-parser",
    "parserOptions": {
        "ecmaVersion": 6,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "rules": {
        "indent": "error",
        "padded-blocks": ["error", "never"]
    }
};

```

**What code were you trying to parse?**

```
`${name}`;
{
    /* TODO Use system attribute name instead of column name */
    1 + 1;
}
```

**What did you expect to happen?**
Padded block with config `['error', 'never']` should not error here

**What happened?**
Eslint fails

```
/mnt/c/Users/reyad.attiyat/Workspace/eslint-test/test-padded-blocks.js
  2:1  error  Block must not be padded by blank lines  padded-blocks
```

Might be related too: https://github.com/eslint/typescript-eslint-parser/issues/216

**Comment by soda0289** [comment]
Same issue as #216

**Comment by soda0289** [comment]
Fixed by https://github.com/eslint/typescript-eslint-parser/pull/219
